### DISCUSSION TITLE
Unable to upgrade from Poster 1.x to v2

### DISCUSSION THREAD

**Discussion by japps20** [postalserver/postal#3013] [Help with using Postal]
Hi there,

I currently have Poster 1.x and following the instruction here (https://docs.postalserver.io/getting-started/upgrade-to-v2) to update to Poster v2, but it just went straight to three thus breaking a few things.

Please is there a specific way to upgrade specifically to Poster v2 and then v3?

**Comment by willpower232** [comment]
That page needs some updates for sure. Would you like to make an issue in https://github.com/postalserver/docs?

Which things broke? Presumably it may just be a matter of following both upgrades or perhaps starting over and importing the data?

**Reply by japps20** [reply]
Not sure what else broke, but the login page seems to have lost all the CSS and some of the message counts were skewed.

If I was to do a fresh install which will install v3 by default, how do I import the data from v1 ? Is there any documentation for this?

**Reply by willpower232** [reply]
The page you were already on ended with some text about how to copy over data to a new installation https://docs.postalserver.io/getting-started/upgrade-to-v2#installing-on-a-new-server-with-existing-data
